libs/response.js

```diff
@@ -2,16 +2,20 @@ export function success(body) {
   return buildResponse(200, body);
 }
 
-export function failure(error) {
+export function failure(error, stackTrace) {
+  const { stage } = process.env;
```

Comment:
we may need to use config.js to be consistent?

Reply:
Agree, forgot about config. Good Catch 🙂